src/org/usfirst/frc/team467/robot/Autonomous/ActionGroup.java

```diff
@@ -118,15 +117,20 @@ public void doIt() {
 		 */
 		public Duration(double duration) {
 			durationMS = duration * 1000;
+			LOGGER.debug("durationMS=" + durationMS);
 		}
 
 		@Override
 		public boolean isDone() {
 			if (actionStartTimeMS < 0) {
 				actionStartTimeMS = System.currentTimeMillis();
+				LOGGER.debug("Action Start Time=" + actionStartTimeMS);
 			}
 
-			return System.currentTimeMillis() > durationMS + actionStartTimeMS;
+			long currentTime = System.currentTimeMillis();
+			LOGGER.debug("currentTime=" + currentTime);
```

Comment:
This is unnecessary because the logger already prints millisecond time with every log message.

Reply:
Fixed. I changed it back to the way it was.